portfolio/src/main/webapp/pagination.js

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+document.addEventListener('DOMContentLoaded', () => {
+  const commentButton = new CommentButton();
+  commentButton.requestAndDisplayNewCommentsPage();
+});
+
+class CommentButton {
+  constructor() {
+    this.currentCommentsPageIndex = 0;
+    this.totalNumberOfPagesRequested = 1;
+    this.lastPage = -1;
+    this.cursorString = null;
+     
+    document.getElementById('prev-comments-button').classList.add("hidden");
+    document.getElementById('end-of-comments-sign').classList.add("hidden");
+   
+    document.getElementById('maxNum').addEventListener('input', () => {
+      document.getElementById('comments').innerHTML = "";
+      document.getElementById('prev-comments-button').classList.add("hidden");
+      if (document.getElementById('next-comments-button').classList.contains("hidden")) {
```

Comment:
instead of checking the classList for the element, can you store a local variable to check against?

Reply:
Oh; i see you were checking it it's there then removing it. it is safe to always call remove; even if the class tag isn't there already.